GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/BasicFileSystemTests.cs

```diff
@@ -918,6 +919,27 @@ public void CreateDirectoryInheritsParentDirectoryAttributes(string parentFolder
             FileSystemRunner.DefaultRunner.DeleteDirectory(parentDirectoryPath);
         }
 
+        [TestCase]
+        [Category(Categories.MacOnly)]
+        public void RunPythonExecutable()
+        {
+            GitProcess.Invoke(this.Enlistment.RepoRoot, "checkout FunctionalTests/20190827");

```

Comment:
I would rename this branch to something like `FunctionalTests/PythonExecutable` just because we have normally used the timestamp branch for the main test branch.